cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -174,7 +227,7 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
           resultTable()
         }
 
-        p("Available privileges for roles can also be output as Cypher commands with the optional `AS COMMAND[S]`.")
+        p("Same as for all privileges, the available privileges for roles can also be output as Cypher commands with the optional `AS COMMAND[S]`.")
```

Comment:
```suggestion
        p("Similar to all other privileges, the available privileges for roles can also be output as Cypher commands with the optional `AS COMMAND[S]`.")
```

Reply:
```suggestion
        p("Similar to the other show privilege commands, the available privileges for roles can also be output as Cypher commands with the optional `AS COMMAND[S]`.")
```

The `all privileges` are referring to the `SHOW PRIVILEGE` command which returns all privileges which made your `all other privileges` slightly odd.

Reply:
In that case, yes, your suggestion is better. Glad that it was caught since that point was missed entirely in the first construction.